app/views/trends/_extraction_warning.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if User.current.admin? %>
```

Comment:
Who is the actual audience here? :thinking:

Reply:
I'm not sure about this. The message you have now is not actionable, right? If you're running 2.1, can you install the plugin? Should it be an upgrade warning instead in the manual?

Reply:
It definitely should be upgrade warning as well... :thinking:
This is just for more visibility..

Reply:
Might be good to display this to all users, the person looking at this page may or may not be the admin